src/Core/src/Platform/iOS/ApplicationExtensions.cs

```diff
@@ -57,14 +58,23 @@ public static void CreatePlatformWindow(this IUIWindowSceneDelegate sceneDelegat
 				dicts.Add(session.UserInfo);
 			if (session.StateRestorationActivity?.UserInfo is not null)
 				dicts.Add(session.StateRestorationActivity.UserInfo);
-			if (connectionOptions.UserActivities is not null)
+			try
 			{
-				foreach (var u in connectionOptions.UserActivities)
+				using var activities = connectionOptions.UserActivities;
```

Comment:
First-point-five level, make sure we also dispose the error set because there may be another set later on.